src/main/java/Event.java

```diff
@@ -0,0 +1,22 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+    private LocalDate eventTime;
+
+    public Event(String description, LocalDate eventTime) {
+        super(description);
+        this.eventTime = eventTime;
+    }
+
+    @Override
+    public String getStatus() {
+        return "[E]" + super.getStatus() + "(at: " + this.eventTime.format(DateTimeFormatter.ofPattern("MMM d yyyy"))
```

Comment:
Kind of a long statement here as well, perhaps I would suggest using StringBuilder for better readability :)